### PR TITLE
Add efm-langserver

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

File: _implementors/servers.md [hunk 1]
```diff
@@ -127,4 +127,5 @@ index: 1
 | YAML (with JSON schemas)| [Adam Voss](https://github.com/adamvoss) | [vscode-yaml-languageservice](https://github.com/adamvoss/vscode-yaml-languageservice) | TypeScript |
 | YAML| [Red Hat Developers](https://github.com/redhat-developer) | [yaml-language-server](https://github.com/redhat-developer/yaml-language-server) | TypeScript |
 | [YANG](https://tools.ietf.org/html/rfc7950)| [Yang tools](https://github.com/yang-tools) | [yang-lsp](https://github.com/yang-tools/yang-lsp) |  XTend |
+| * | [mattn](https://github.com/mattn) | [efm-langserver](https://github.com/mattn/efm-langserver) | Any |
```
Comment (mattn) [inline _implementors/servers.md]:
```suggestion
| * | [mattn](https://github.com/mattn) | [efm-langserver](https://github.com/mattn/efm-langserver) | [Go](https://golang.org/) |

```

**Comment by mattn** [comment]
Oops. Sorry. I broke something?

https://microsoft.github.io/language-server-protocol/implementors/servers/

**Comment by rcjsuen** [comment]
@mattn It looks okay to me. What are you seeing?

**Comment by dbaeumer** [comment]
Actually the table rendering (grid lines) broke

capture

**Comment by mattn** [comment]
If my memory is certain, when I saw the list in a few weeks ago, there was a border on the table.

**Comment by dbaeumer** [comment]
@mattn correct it should look like this: 

capture


Let me have a look to find out what happened.

**Comment by dbaeumer** [comment]
Fixed it. Was the extra newline before `{: .table .table-bordered .table-responsive}`

**Comment by mattn** [comment]
Thank you!

**Pull request merged**